roles/logs/tasks/logs_to_hdd.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Create probably missing log folders on external HDD
+  file:
+    path: "{{ bsf_log_folder }}/{{ item.name }}"
+    owner: "{{ item.owner }}"
+    group: "{{ item.group }}"
+    mode: 0775
+    state: directory
+  with_items:
+    - name: nginx
+      owner: root
+      group: adm
+    - name: uwsgi
+      owner: ideascube
+      group: ideascube
+    - name: samba
+      owner: root
+      group: adm
+    - name: telegraf
+      owner: telegraf
```

Comment:
At this stage the user `telegraf` is unknown. Telegraf role should be executed before logs role